CheckYourEligibility-Parent/Views/Home/SchoolList.cshtml

```diff
@@ -0,0 +1,71 @@
+﻿@model CheckYourEligibility_FrontEnd.ViewModels.SchoolListViewModel
+@{
+    ViewData["Title"] = "Private Beta School List";
+    ViewData["Layout"] = "two_thirds";
+}
+<a class="govuk-back-link" href="@Url.Action("Index", "Home")" onclick="history.back(); return false;">Back</a>
+<script>document.body.className += ' js-enabled' + ('noModule' in HTMLScriptElement.prototype ? ' govuk-frontend-supported' : '');</script>
```

Comment:
should this really be here? Where is it from?

Reply:
This is stuff from prototype, tried to strip out the parts that weren't necessary but think I missed this